notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2907 @@
+{
```

Comment:
<blockquote>flakiness score dictionary using Flip Flakiness with Optimal Distance method is:  {}</blockquote><p>Is this the expected behavior for this method? '{}` is not score. Shouldn't it be 0.0? </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
flip flakiness with optimal distance returns a dictionary where the flakiness score is above a certain threshold.
we didn't get a single range where the flakiness score is above 0.

I added this to the documentation. Also, added the number of flaky tests detected for more understanding.